src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -263,6 +273,10 @@ export default {
 			return generateUrl('/settings/user/notifications')
 		},
 
+		themingUrl() {
+			return generateUrl('/settings/user/theming')
```

Comment:
Need to check if the server version is "high-enough"?

Reply:
29.0.4 you mean?

Reply:
I guess so?